### DISCUSSION TITLE
https://docsaid.org/en/blog/update-docusaurus-to-3-6-0/

### DISCUSSION THREAD

**Discussion by giscus** [DocsaidLab/website#11] [Announcements]
# https://docsaid.org/en/blog/update-docusaurus-to-3-6-0/

Troubleshooting issues during the update process

https://docsaid.org/en/blog/update-docusaurus-to-3-6-0/

**Comment by frankelinli** [comment]
请问您的翻译版本，是自动翻译吗？每篇文章都有英文版。很厉害。在向您的网站学习

**Reply by zephyr-sh** [reply]
在 docusaurus 的教學文件中有提到自動翻譯的相關功能，但根據我的經驗，大多品質不佳。從 docusaurus 所提供的多國語言的「中文」版就可以看得出來，讓人憂慮。

因此我的對策是：先把中文文章寫完之後，「逐段」請 ChatGPT 翻譯，並檢視其成果後再手動校對修改。（很繁瑣）

但考慮到 LLM 大多基於英語進行訓練，中文語料相對少，因此在應對成語或是一些我們所使用的幽默語句時，會給出不正確的翻譯。在實際使用中，我們就可以頻繁看到許多不正確的理解和句子。因此為了維持一定的翻譯品質，相對繁瑣的步驟還是不能少的。

結論是你只要找個品質不錯的 LLM，就可以輔助你完成翻譯的工作。

**Reply by frankelinli** [reply]
感谢给出的翻译思路。我也在逐步给网站加上多语言功能。
